gemd/entity/template/tests/test_process_template.py

```diff
@@ -39,3 +42,35 @@ def test_allowed_labels():
     proc_template = ProcessTemplate(name="test template", allowed_labels=allowed_labels)
     for label in allowed_labels:
         assert label in proc_template.allowed_labels
+
+
+def test_passthrough_bounds():
+    """Test that unspecified Bounds are accepted and set to None"""
+    template = ProcessTemplate('foo', conditions=[
+        (LinkByUID('1', '2'), None),
+        [LinkByUID('3', '4'), None],
+        LinkByUID('5', '6'),
+        ConditionTemplate('foo', bounds=IntegerBounds(0, 10)),
+    ])
+    assert len(template.conditions) == 4
+    for _, bounds in template.conditions:
+        assert bounds is None
+    copied = loads(dumps(template))
```

Comment:
```suggestion
    copied = copy(template)
```
As above

Reply:
I actually kinda like `loads(dumps(` here because it guarantees full round-trip serialization (which is what I wanted to test), whereas `copy` could conceivably be implemented another way.

Reply:
I see where you are going.  Acceptable.